server/resources/request_new.py

```diff
@@ -0,0 +1,50 @@
+from flask_restful import Resource, reqparse
+from flask_jwt_extended import (jwt_required, get_jwt_identity)
+
+from server.models.review_model import ReviewModel
+#from server.run import queue_object, redis_instance
+from server.tasks.find_reviewer_task import find_reviewer
+from server.extensions import queue_object
+
+
+import time
+import json
+
+class RequestNew(Resource):
+    @jwt_required
+    def post(self):
+        # create review obj, add it to db
+        # create task to find user for request
+        parser = reqparse.RequestParser()
+        parser.add_argument("title", help="This field cannot be blank", required=True)
+        parser.add_argument("code", help="This field cannot be blank", required=True)
+        parser.add_argument("lang_level", help="This field cannot be blank", required=True)
+        data=parser.parse_args()
+        user_id = get_jwt_identity()
+        title = data["title"]
+        messages_data = {1: data["code"]} 
```

Comment:
Yea I'd prefer messages to probably be another model - one to many model (https://code.tutsplus.com/articles/sql-for-beginners-part-3-database-relationships--net-8561)